src/main/java/synapseawsconsolelogin/Auth.java

```diff
@@ -450,4 +472,17 @@ private String getSSMParameter(String name) {
 		}
 	}
 
+	public String initAppVersion() {
+		Properties gitProps = loadProperties(GIT_PROPERTIES_FILENAME);
+		if (! (gitProps.containsKey(GIT_COMMIT_TIME_KEY)) && (gitProps.containsKey(GIT_COMMIT_ID_DESCRIBE_SHORT_KEY))) {
```

Comment:
```suggestion
		if (! (gitProps.containsKey(GIT_COMMIT_TIME_KEY)) && ! (gitProps.containsKey(GIT_COMMIT_ID_DESCRIBE_SHORT_KEY))) {
```

Reply:
I wanted !(x&&y) but you're right the parens are screwed up... Good catch!